replay: JS CDN docs (#5932)
getsentry/sentry-docs: src/platforms/javascript/common/install/cdn.mdx

});
```

## Session Replay Bundle

```html {tabTitle: CDN}
<script
  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/replay.min.js"
  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'replay.min.js', 'sha384-base64') }}"
  crossorigin="anonymous"
></script>
```

<Note>

This is an addition to the bundles mentioned above for errors and tracing. 

> </Note>

Reviewer: I don't understand what we're saying with this note. "Additional" just communicates this is another available bundle, which seems like it's obvious. Do you want to say that it's new?

Author: @imatwawana on the bundle above it's not an addition but rather a replacement. I was trying to emphasize that this one is not to be a replacement to `bundle.js` or `tracing.bundle.js`. You need either one of those *plus* this one. 
Any other way to suggest that? Or should I leave this out?

Reviewer: Ah okay. I would actually suggest putting this in plain text:
```suggestion
Use this bundle **in addition to** one of the bundles (for errors and tracing) above:
```

> Please refer to <PlatformLink to="/session-replay/">the Session Replay documentation</PlatformLink> for more information.

Reviewer: ```suggestion
Please refer to our <PlatformLink to="/session-replay/">Session Replay documentation</PlatformLink> to learn more.
```
